### DISCUSSION TITLE
More beginner-oriented documentation?

### DISCUSSION THREAD

**Discussion by ghost** [framesurge/perseus#209] [General]
How to explain words and phrases like "hydration", "checkpoint", "revalidation", "build paths", etc, to people who don't know anything about HTML and JavaScript?

**Comment by erlend-sh** [comment]
This is a web framework, so it’s fair to assume some foundational knowledge about HTML and JavaScript.

**Reply by arctic-hen7** [reply]
Honestly, I actually think it's perfectly reasonable to define these terms at a more basic level. The very long term goal is to use Wasm over JS, and I think that will involve new users working solely with Wasm. It's a small section of the user base for now, but I think this should be something to keep in mind.

I wonder about an independent project that introduces terms like these in a framework and language neutral way?

**Comment by arctic-hen7** [comment]
How about you make a file in `docs/next/` with a list of all the terms you're unclear about, and then make a PR for that. I can then put in definitions to exactly what you want, and we can merge it into the docs for everyone in the future!

(If you need any guidance on this process, I'm happy to help, just let me know.)

**Reply by ghost** [reply]
What does "rx" or "Rx" mean in #[perseus::template_rx], IndexPropsRx, #[perseus::make_rx(IndexPropsRx)], etc?

**Reply by arctic-hen7** [reply]
Rx is short for reactive.
